app/models/reminder.js

```diff
@@ -3,5 +3,5 @@ import DS from 'ember-data';
 export default DS.Model.extend({
   title: DS.attr('string'),
   body: DS.attr('string'),
-  date: DS.attr('date'),
+  date: DS.attr('string'),
```

Comment:
Because @martensonbj said to focus on getting it as a string for now since ember is a nightmare with DS.attr('date')